dlrs/main/classes/RollupController.cls

```diff
@@ -60,6 +59,7 @@ public with sharing class RollupController {
   public ApexClass RollupParentTriggerTest { get; private set; }
 
   public Integer deployCount;
+  public String apiVersion;
```

Comment:
As before, I see a private property that is testvisible instead of a public one. Would we want to set this value from outside (outside of test context?)